maintainers/maintainer-list.nix

```diff
@@ -20743,6 +20743,12 @@
     githubId = 20863431;
     name = "Prithak S.";
   };
+  priyaananthasankar = {
+    name = "Priya Ananthasankar";
+    github = "priyaananthasankar";
+    github = 10415876;
+    email = "priyagituniverse@gmail.com";
+  };
```

Comment:
This needs to be removed from the commit
ec9828d1ef66e19e723c725b12c8297acc859448